drivers/include/sdp3x.h

```diff
@@ -49,6 +49,7 @@ typedef struct {
     i2c_t i2c_dev;                  /**< I2C device which is used */
     uint8_t i2c_addr;               /**< I2C address */
     gpio_t irq_pin;                 /**< IRQ pin to check data ready */
+    bool irq_device;                /**< IRQ pin connected */
```

Comment:
No need for an extra variable - you can check for `irq_pin != GPIO_UNDEF` to see if a IRQ pin is defined.

Reply:
fixed